src/schemas/json/package.json

```diff
@@ -99,6 +99,10 @@
         "default": {
           "$ref": "#/definitions/packageExportsEntryOrFallback",
           "description": "The module path that is resolved when no other export type matches."
+        },
+        "types": {
+          "$ref": "#/definitions/packageExportsEntryOrFallback",
+          "description": "The module path that is resolved for TypeScript types when this specifier is imported as an ECMAScript module using an `import` declaration or the dynamic `import(...)` function."
```

Comment:
from discussion with @andrewbranch -- drop the last half of the sentence and stop at "is imported". And it should be listed before other conditions.